### PR TITLE
updates from Marc

### PR DESCRIPTION
**Pull request by mmosko**
Mostly changed intro

### CODE REVIEW

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 1]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
-Ephemeral sessions similar to those enabled by TLS 1.3 {{TLS13}},
-QUIC {{QUIC}}, and DTLS 1.2 {{DTLS12}} are needed for some
-CCN exchanges between consumers and producers. Currently, there does not exist
-a standard way to establish these sessions. Thus, the primary goal of
-the CCNxKE protocol is to provide privacy and data integrity between two CCN-enabled
-peers (e.g., a consumer and producer engaged in session-based communication). It
+CCNx Key Exchange (CCNxKE) establishes ephemeral forward secure keys between two peers,
+called the consumer (client) and producer (server).  The underlying cryptography of
+CCNxKE is similar to TLS 1.3, though there are some protocol changes due to the ICN nature
+of CCNxKE.  CCNxKE also supports the concept of a MoveToken, which allows then authenticating
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: which allows "the"

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 2]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+producer to shift a session to one (or more) co-operating replicas.  
+
+CCNxKE does not specify how the keys are used.  It only specifies how to derive the traffic
+secret that could be used to encrypt/decrypt data.  The draft [draft-wood-icnrg-tlvencap] specifies
+one way to use the traffic secret to carry out communications.
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: "... to carry out communication in a session."

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 3]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+CCNx Key Exchange (CCNxKE) establishes ephemeral forward secure keys between two peers,
+called the consumer (client) and producer (server).  The underlying cryptography of
+CCNxKE is similar to TLS 1.3, though there are some protocol changes due to the ICN nature
+of CCNxKE.  CCNxKE also supports the concept of a MoveToken, which allows then authenticating
+producer to shift a session to one (or more) co-operating replicas.  
+
+CCNxKE does not specify how the keys are used.  It only specifies how to derive the traffic
+secret that could be used to encrypt/decrypt data.  The draft [draft-wood-icnrg-tlvencap] specifies
+one way to use the traffic secret to carry out communications.  
+Annex A also sketches out an example CCNx protocol for exchangig encrypted messages,
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: "for exchanging"

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 4]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+though it is not part of this standard.
+Other protocols may use CCNxKE.  
+For example, a producer and replica may use CCNxKE to establish a shared key to use
+in Move Tokens.  Two routers may use CCNxKE to establish MACSEC keys.  A consumer and publisher could establish
+a symmetric key while on-line then publish content later for an off-line consumer.
+The use of CCNxKE does not, in short, require the use of a TLS-like transport protocol.
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: "In short, the use of CCNxKE is not limited to a TLS-like secure transport protocol."

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 5]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+
+CCNxKE allows upper-layer data to be returned in Round 3, like TLS 1.3.  In this sense, one can achieve 3 RTT
+(worst case) or 1 RTT (best case) communcations.  The data put in this response is up to the 
+protocol using CCNxKE and may or may not be used.
+
+CCNxKE is not a substitue for
+data security, such as Content Object provenance via signatures, group encryption of cached objects,
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: "for data authenticity"

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 6]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+or DRM protections.  CCNxKE only creates a private, ephemeral tunnel between a consumer
+and a producer.  CCNxKE expects that the encrypted communications protocol still carries
+normal CCNx packets with normal CCNx attributes such as signatures and hash chains.
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: Just drop "and hash chains"

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 7]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+
+Some types of ICN communications require emphemeral, forward secure encryption.
+Typical examples are on-line banking, real-time voice, or on-line shopping.
+Other applications may need different types of encryption and thus not use CCNxKE.
+There is currently no standard way for CCNx peers to excahnge emphemeral, forward
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: typo "exchange"

File: draft-wood-icnrg-ccnxkeyexchange-01.md [hunk 8]
```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
...
+secure keys, thus this RFC specifies the standard mechanism that should be used
+buy all CCNx peers for such keys. CCNxKE
```
Comment (chris-wood) [inline draft-wood-icnrg-ccnxkeyexchange-01.md]: buy -> by

**Comment by mmosko** [comment]
Updated with fixes to comments.

**Pull request merged**